src/py/flwr/server/strategy/aggregate.py

```diff
@@ -41,6 +42,31 @@ def aggregate(results: List[Tuple[NDArrays, int]]) -> NDArrays:
     return weights_prime
 
 
+def aggregate_inplace(results: List[Tuple[ClientProxy, FitRes]]) -> NDArrays:
```

Comment:
I'll have to change then that in `aggregate`, since otherwise, passing `results` as `List[Tuple[NDArrays, int]]` to `aggregate_inplace` would require a copy of all updates sent by clients.